geoextent/lib/extent.py

```diff
@@ -2,18 +2,17 @@
 import os
 import threading
 import zipfile
+from osgeo import ogr
+from osgeo import gdal
 
 from . import handleCSV
-from . import handleGeojson
-from . import handleGeotiff
-from . import handleShapefile
+from . import handleVector
+from . import handleRaster
 from . import helpfunctions as hf
 
 logger = logging.getLogger("geoextent")
 
-modulesSupported = {'geojson': handleGeojson, 'json': handleGeojson, 'csv': handleCSV,
-                    'shp': handleShapefile, 'dbf': handleShapefile, 'geotiff': handleGeotiff, 'tif': handleGeotiff}
-
+special_modules = {'CSV': handleCSV}
```

Comment:
Done ! I'll include this in the next commit. We already have a checkValidity() function for each module. I'll modify the checkValidity() output for (True/False) in case that handle supports (or not) the file.